circus-train-avro/src/main/java/com/hotels/bdp/circustrain/avro/util/SchemaCopier.java

```diff
@@ -13,35 +13,33 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
+
 package com.hotels.bdp.circustrain.avro.util;
 
 import static com.google.common.base.Preconditions.checkNotNull;
-
 import static com.hotels.bdp.circustrain.avro.util.AvroStringUtils.fileName;
 
 import java.io.IOException;
 import java.nio.file.Files;
-
+import org.apache.hadoop.conf.Configuration;
```

Comment:
NABD but there should be a line break between java.* and org.* imports (like it was before)